modules/ROOT/pages/procedures.adoc

```diff
@@ -1884,63 +1884,69 @@ The procedures deprecated in Neo4j 5 will be removed in the next major release o
 | label:no[]
 | label:yes[]
-| label:new[Introduced in 5.13] label:beta[] label:deprecated[Deprecated in 5.17]
+| label:beta[] label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
 Replaced by: xref:procedures.adoc#procedure_db_cdc_earliest[`db.cdc.earliest()`]
 
 | xref:procedures.adoc#procedure_cdc_query[`cdc.query()`]
 | label:no[]
 | label:yes[]
-| label:new[Introduced in 5.13] label:beta[] label:admin-only[] label:deprecated[Deprecated in 5.17]
+| label:beta[] label:admin-only[] label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
 Replaced by: xref:procedures.adoc#procedure_db_cdc_query[`db.cdc.query()`]
 
 | xref:procedures.adoc#procedure_db_create_setVectorProperty[`db.create.setVectorProperty()`]
 | label:yes[]
 | label:yes[]
-| label:new[Introduced in 5.11] label:beta[] label:deprecated[Deprecated in 5.13] Replaced by: xref:procedures.adoc#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty()`]
+| label:beta[] label:deprecated[Deprecated in Cypher 5] label:removed[Removed in Cypher 25] +
+Replaced by: xref:procedures.adoc#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty()`]
 
-// New in 4.2
-// com.neo4j.causaulclustering.discovery.procedures.ReadReplicaToggleProcedure
 | xref:procedures.adoc#procedure_dbms_cluster_readreplicatoggle[`dbms.cluster.readReplicaToggle()`]
 | label:no[]
 | label:yes[]
-| label:admin-only[] label:deprecated[Deprecated in 5.6]. +
+| label:admin-only[] label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
 Replaced by: xref:procedures.adoc#procedure_dbms_cluster_secondaryreplicationdisable[`dbms.cluster.secondaryReplicationDisable()`].
 
 | xref:procedures.adoc#procedure_dbms_cluster_routing_getroutingtable[`dbms.cluster.routing.getRoutingTable()`]
 | label:yes[]
 | label:yes[]
-| label:deprecated[Deprecated in 5.21]. +
+| label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
 Replaced by: xref:procedures.adoc#procedure_dbms_routing_getroutingtable[`dbms.routing.getRoutingTable()`].
 
 | xref:procedures.adoc#procedure_dbms_cluster_uncordonServer[`dbms.cluster.uncordonServer()`]
 | label:no[]
 | label:yes[]
-| label:deprecated[Deprecated in 5.23]. +
-Before Neo4j 5.23, the procedure can be run only with the `Admin` privileges. +
+| label:deprecated[Deprecated in Cypher 5] label:removed[Removed in Cypher 25] +
 Replaced by xref:clustering/server-syntax.adoc#server-management-syntax[`ENABLE SERVER`].
 
 | xref:procedures.adoc#procedure_dbms_setDatabaseAllocator[`dbms.setDatabaseAllocator()`]
 | label:no[]
 | label:yes[]
 | label:admin-only[] label:deprecated[Deprecated in 5.23]
 
-// New in 4.1
 | xref:procedures.adoc#procedure_dbms_upgrade[`dbms.upgrade()`]
 | label:yes[]
 | label:yes[]
-| label:admin-only[] label:deprecated[Deprecated in 5.9]
+| label:admin-only[] label:deprecated[Deprecated in Cypher 5] label:removed[Removed in Cypher 25]
```

Comment:
```suggestion
| label:admin-only[] label:deprecated[Deprecated since Neo4j 5.9] label:available[Available in Cypher 5] label:removed[Removed in Cypher 25]
```